doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -68,12 +93,14 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.vim.01` | General | The Architecture **must** allow infrastructure resource sharing among several service operation teams to avoid per VNF vertical solutions. |
-| `req.vim.02` | General | The Architecture shall/must xxx. |
-| `req.vim.03` | General | The Architecture shall/must xxx. |
-| `req.vim.04` | General | The Architecture shall/must xxx. |
-| `req.vim.05` | General | The Architecture shall/must xxx. |
-| `req.vim.06` | General | The Architecture shall/must xxx. |
+| `req.vim.01` | General | The Architecture **must** allow infrastructure resource sharing. |
+| `req.vim.02` | General | The Architecture **should** support deployment of OpenStack components in containers. |
+| `req.vim.03` | General | The Architecture **must** allow VIM to discover and manage NFVI resources. |
+| `req.vim.04` | General | The Architecture **must** support Enhanced Platform Awareness (EPA). |
+| `req.vim.05` | General | The Architecture **must** include image repository management and provisions component. |
+| `req.vim.06` | General | The Architecture **must** allow orchestration solutions to be integrated with VIM. |
```

Comment:
Shouldn't be the integration solely through the exposed APIs? Proposal to delete this requirement.

Reply:
The point here is that quite often in real life the implementations of VIMs do not provide such APIs.  I propose to qualify that this integration should be done by standard ETSI APIs.

Reply:
@walterkozlowski is there a need to re-word?